api/apps/api/src/modules/scenarios/planning-units/scenario-planning-units-protection-status-calculator-service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { DbConnections } from '@marxan-api/ormconfig.connections';
+import { ScenariosPlanningUnitGeoEntity } from '@marxan/scenarios-planning-unit';
+import { Scenario } from '../scenario.api.entity';
+
+@Injectable()
+export class ScenarioPlanningUnitsProtectedStatusCalculatorService {
```

Comment:
This service could be useful for geoprocessing as well (when changing inclusion status).

Reply:
yep, we should move this to a shared library in a second pass

Reply:
Fancy creating a relevant task in tracking system?

Reply:
https://vizzuality.atlassian.net/browse/MARXAN-649